test.sh

```diff
@@ -41,10 +41,12 @@ done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
 rm -f .libs/tests-linux64.*
 rock -q +-Wall $ARGS $FLAGS $FEATURES $TESTS_USE_FILE
-if [[ $? -eq 0 && -f ./Tests ]]; then
+exitcode=$?
+if [[ $lol -eq 0 && -f ./Tests ]]; then
 	./Tests
+	exitcode=$?
 fi
-if [[ !( $? == 0 ) ]]
+if [[ !( $exitcode == 0 ) ]]
```

Comment:
Why do we do this? Why not just `exit $exitcode`? @thomasfanell , you wrote this block eight months ago, do you know?